pkg/settlement/swap/swap_test.go

```diff
@@ -600,6 +603,9 @@ func TestHandshakeNewPeer(t *testing.T) {
 }
 
 func TestHandshakeWrongBeneficiary(t *testing.T) {
+
+	t.Skip()
```

Comment:
why do you skip this?

Reply:
this test can actually be removed after @aloknerurkar 's PR